tools/tasks/css_task.ts

```diff
@@ -0,0 +1,11 @@
+import { Task } from './task';
+
+export abstract class CssTask extends Task {
+
+  precondition(files: String[]) {
+    return files.reduce((a, f) => {
```

Comment:
couldn't all those reduce calls be simplified with `files.some(f => f.endsWith('.css') || ...)`?